float64 ts
EcatSlaveInfo[] slaves